### PR TITLE
JOSS paper draft

### PR DESCRIPTION
**Pull request by batpigandme**

### CODE REVIEW

**Comment by hadley** [comment]
I pushed feedback into your PR. Can you let me know if it's not obvious want I want, but otherwise take another pass at it?

**Comment by hadley** [comment]
@batpigandme can you please take another pass? It needs proofing reading, I've inserted a few specific requests in comments, and I'd appreciate any ideas on the new philosophy section.

File: vignettes/paper.Rmd [hunk 1]
```diff
@@ -28,109 +111,102 @@ org %>% purrr::map_chr("login") %>% clipr::write_clip()
...
+The scope of the tidyverse covers the repeated tasks at the heart of every data science project, including data import, wrangling, and visualisation. We expect that almost every project will use multiple packages from outside of the tidyverse: the goal is to provide tooling for the most common challenges, not to solve every possible problem. In particular, the tidyverse doesn't include tools for statistical modelling or communication with others. These are both critical for data science projects, but big enough domains that they merit separate treatment.
 
+This paper describes the tidyverse package, the components of the tidyverse, and some of the underlying philosophy.
 
+# Tidyverse package
 
-This paper documents the tidyverse package which provides a convenient way of downloading all tidyverse packages with a single R command:
+As well as a being a collection of packages, the tidyverse itself is also a package, which provides a convenient way of downloading all tidyverse packages with a single R command:
```
Comment (jennybc) [inline vignettes/paper.Rmd]:
```suggestion
As well as being a collection of packages, the tidyverse itself is also a package, which provides a convenient way of downloading all tidyverse packages with a single R command:
```

**Comment by hadley** [comment]
Oh I merged this into a branch?

**Pull request merged**